crates/store/re_tf/src/transform_resolution_cache.rs

```diff
@@ -61,6 +62,14 @@ impl Default for TransformResolutionCache {
     }
 }
 
+impl SizeBytes for TransformResolutionCache {
+    fn heap_size_bytes(&self) -> u64 {
```

Comment:
I think all of these `heap_size_bytes` should do self deconstruct via `Self { .. } = self;` because I'm fairly sure that already my currently open PRs may add fields that are gonna get missed otherwise